presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -97,6 +99,7 @@ public class QueryInfo
     private final Set<String> windowsFunctions;
     // Using a list rather than map, to avoid implementing map key deserializer
     private final List<CanonicalPlanWithInfo> planCanonicalInfo;
+    private Map<PlanNodeId, PlanNode> planIdNodeMap;
```

Comment:
PlanNode is needed as input in calculating the output size of a node https://github.com/prestodb/presto/blob/d541525b28f1da0f99a20e50482ecae4189bb57d/presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimate.java#L128